pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/ParticipantDao.sql.stg

```diff
@@ -55,13 +55,26 @@ select en.user_study_enrollment_id as enrollment_id,
        mp.phone                      as m_phone,
        mp.legacy_guid                as m_legacy_guid,
        mp.street                     as m_street,
-       (select institution_type_code from institution_type where institution_type_id = mp.institution_type_id) as m_institution_type_code
+       (select institution_type_code from institution_type where institution_type_id = mp.institution_type_id) as m_institution_type_code,
+       i.invitation_id as i_invitation_id,
+       it.invitation_type_code as i_invitation_type_code,
+       i.user_id as i_user_id,
+       i.invitation_guid as i_invitation_guid,
+       i.study_id as i_study_id,
+       i.created_at as i_created_at,
+       i.voided_at as i_voided_at,
+       i.verified_at as i_verified_at,
+       i.accepted_at as i_accepted_at,
+       i.contact_email as i_contact_email,
+       i.notes as i_notes
   from user_study_enrollment as en
   join user as u on u.user_id = en.user_id
   left join user_profile as up on up.user_id = u.user_id
   left join default_mailing_address as def_addr on def_addr.participant_user_id = u.user_id
   left join mailing_address as addr on def_addr.address_id = addr.address_id
   left join user_medical_provider as mp on mp.user_id = u.user_id and mp.umbrella_study_id = en.study_id
+  left join invitation as i on i.user_id = mp.user_id and i.study_id = en.study_id
+  left join invitation_type as it on i.invitation_type_id = it.invitation_type_id
```

Comment:
I'm a bit worried about adding a left join here. A participant has only one profile, one default address, and multiple medical providers. So the query here is meant to product a result set that effectively create a one-to-many relation as "one participant to many medical providers".

But if we add in this left join here, then it expands the result set to "one participant to many medical providers to many invitations", which might produce duplicate results.